static/app/views/alerts/rules/metric/details/anomalyDetectionFeedbackBanner.tsx

```diff
@@ -50,6 +52,15 @@ export default function AnomalyDetectionFeedbackBanner({
         level: 'info',
         message: 'Anomaly Detection Alerts Banner Feedback',
       });
+      if (!anomalyCorrectlyIdentified && openForm) {
+        openForm({
```

Comment:
A tad more descriptive about the intent at call site, so you dont have to go look at what form it open :)
```suggestion
        openFeedbackForm({
```